modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/idp/mgt/IdentityProviderMgtServiceTestCase.java

```diff
@@ -360,7 +361,106 @@ public void testGetAllIdPs() throws Exception {
         }
     }
 
-    @Test(priority = 4, groups = "wso2.is", description = "test getEnabledAllIdPs operation")
+    @Test(priority = 4, groups = "wso2.is", description = "Add Idp for testing filter")
+    public void addIdpForPagination() throws Exception {
```

Comment:
since we already have a test method for addIDP, no need of this method. Use a private method to add another IDP and use that in testGetAllPaginatedPaIdPs